tests/install/openqa_webui.pm

```diff
@@ -102,4 +102,13 @@ sub run {
     clear_root_console;
 }
 
+sub post_fail_hook ($self) {
+    get_log 'tail -n 400 /var/log/zypper.log' => 'zypper.log.txt';
+    get_log 'ls -la /var/cache/zypp/raw/openQA/repodata/' => 'repodata.log.txt';
+    # "-vvv" added due to https://progress.opensuse.org/issues/163112
+    assert_script_run('zypper -n --gpg-auto-import-keys -vvv ref', timeout => 600);
```

Comment:
this has the potential to abort the post_fail_hook (and thus not reaching `$self->SUPER::post_fail_hook`)

Reply:
Yes. But possibly the parent post fail hook does not tell us more then. I guess we should be ok to see what happens about that. We monitor the execution of all openQA-in-openQA tests in the tools team and I expect that we see the effect.

Reply:
we don't get all the logs uploaded without the parent fail_hook - sometimes they can have nice information